frontend/src/templates/Field/YesNo/YesNoField.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { Text } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+
+import { BasicField } from '~shared/types/field'
+
+import Button from '~components/Button'
+
+import {
+  YesNoField as YesNoFieldComponent,
+  YesNoFieldProps,
+  YesNoFieldSchema,
+} from './YesNoField'
+
+export default {
+  title: 'Templates/Field/YesNoField',
+  component: YesNoFieldComponent,
+  decorators: [],
+  parameters: {
+    docs: {
+      // Required in this story due to rhf conflicting with Storybook somehow.
```

Comment:
can we specify that rhf is react-hook-form hahahahaha I don't think it's a well-understood acronym